app/billing_client.go

```diff
@@ -192,6 +192,7 @@ func (c *BillingClient) SendPaymentRequiredEmail(ctx context.Context, customerOr
 }
 
 func usageCostTypeFromProto(costType pb.UsageCostType) UsageCostType {
+	//nolint:exhaustive
```

Comment:
?

Reply:
The linter was complaining about this but I don't know how it got merged 🤷🏾‍♂️